backend/src/seed/seed-db.js

```diff
@@ -106,27 +104,27 @@ import { gql } from '../jest/helpers'
     ])
 
     const [racoon, rabbit, wolf, bear, turtle, rhino] = await Promise.all([
-      f.create('Badge', {
+      factory.create('Badge', {
         id: 'indiegogo_en_racoon',
         icon: '/img/badges/indiegogo_en_racoon.svg',
       }),
-      f.create('Badge', {
+      factory.create('Badge', {
         id: 'indiegogo_en_rabbit',
         icon: '/img/badges/indiegogo_en_rabbit.svg',
       }),
-      f.create('Badge', {
+      factory.create('Badge', {
         id: 'indiegogo_en_wolf',
-        icon: '/img/badges/indiegogo_en_wolf.svg',
+        icon: '/img/badges/indiegogo_en_wolfactory.svg',
```

Comment:
@mattwr18 what is a `wolfactory`? :wolf: :factory: 

:laughing:

Reply:
haahahah global find and replace :rofl: